python/interpret_community/widget/js/main.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import { ExplanationDashboard } from 'interpret-dashboard';
+import { NewExplanationDashboard } from 'interpret-dashboard';
```

Comment:
Hopefully at some point in the near future we can change this to be something other than NewExplanationDashboard

Reply:
Yes, I was thinking we could rename the old one "LegacyExplanationDashboard" so if anyone does insist on using it, its available in the package, but v2 becomes the default one. There is no breaking change on the interface, one can be subbed in for the other at this point.